repos/dir.targets

```diff
@@ -167,6 +167,12 @@
                 PathToAttribute="%(_PossibleCliVersionJsonPath.Identity)"
                 NewAttributeValue="$(SDK_VERSION)"
                 SkipUpdateIfMissingKey="true" />
+    <!-- we always produce an arcade with version 1.0.0-dev -->
+    <UpdateJson JsonFilePath="$(GlobalJsonFile)"
```

Comment:
The source built sdk resolver should tell msbuild what version to use... is this needed now for some other reason?

Reply:
No, this was an early attempt.  Removed.